api/swagger.yml

```
@@ -69,6 +69,17 @@ components:
       description: delimiter used to group common prefixes by
       schema:
         type: string
+    
+    IfNonMatch:
+      in: header
+      name: If-None-Match
+      description: |
+        Currently supports only "*" to allow uploading an object only if one doesn't exist yet.
```

I prefer a positive formulation, such as e.g.
```suggestion
        Set to "*" to atomically allow the upload only if the key has no object yet.  Other
        values are not supported.
```